### DISCUSSION TITLE
Add the menu on the Telemetry page of nano.community

### DISCUSSION THREAD

**Discussion by 1401Ger** [mistakia/nano-community#91] [Proposals]
Issue: When navigating to the Telemetry page (https://nano.community/telemetry) the (floating) menu button in the top left corner is missing. This makes navigation messy since users have to either scroll all the way down to find the (fixed) menu or click "back" in their browser. It is also inconsistent with all other pages that have this menu.

Proposed fix: Add menu button to Telemetry page.

**Comment by mistakia** [comment]
nice catch — converting this to an issue to be addressed (https://github.com/mistakia/nano-community/issues/94)
